packages/api/src/external/ehr/athenahealth/command/get-patients-from-appointments.ts

```diff
@@ -224,10 +222,51 @@ async function getPatientIdOrFail({
       triggerDq,
     });
   } catch (error) {
-    const msg = `Failed to find or create patients. cxId ${cxId} athenaPracticeId: ${athenaPracticeId}. athenaPatientId: ${athenaPatientId}. Cause: ${errorToString(
+    const msg = `Failed to find or create patients. cxId ${cxId} athenaPracticeId ${athenaPracticeId} athenaPatientId ${athenaPatientId}. Cause: ${errorToString(
```

Comment:
ditto re logs

Reply:
same this as above: This is for sentry. The log message would appear in AWS logs, but not sentry. Normally you'd pass an "extras" object with these values (and exclude them from the msg) but I'm aggregating all the errors into a single sentry alert, so I need to make sure they're in the message itself.

Reply:
This one still needs the detailed info as the incoming log is the generic global one.